### PR TITLE
added draytek vigor mime type

### PR DESCRIPTION
**Pull request by LukasEpheser**

### CODE REVIEW

File: fact_helper_file/mime/custom_mime_firmware_containers [hunk 1]
```diff
@@ -714,3 +714,13 @@
...
+
+# Draytek Vigor Firmware
+# Source: https://github.com/q3k/vraytekdigor/blob/master/build/fwtool/fwimage.go
+0       string  2RDH    Draytek Vigor Firmware Container
+!:mime firmware/draytek
+>4      ubelong x       header size: %d bytes,
+>8      ubelong x       file size: %d bytes,
+>12     ubelong x       crc32: 0x%X,
+>80     ubelong x       kernel size: %d bytes,
+>84     ubelong x       squashfs size: %d bytes,
```
Comment (rhelmke) [inline fact_helper_file/mime/custom_mime_firmware_containers:726]: 🤔 Something's not quite right here. Those are all unsigned integers, shouldn't you use a `%u` format string placeholder then?

**Review by rhelmke — CHANGES_REQUESTED** [review]

File: fact_helper_file/mime/custom_mime_firmware_containers [hunk 2]
```diff
@@ -714,3 +714,13 @@
 # SDN unpacker (artificial)
 0	string \x53\x44\x4E	FKIE SDN Unpacker
 !:mime	firmware/sdn
+
+# Draytek Vigor Firmware
+# Source: https://github.com/q3k/vraytekdigor/blob/master/build/fwtool/fwimage.go
+0       string  2RDH    Draytek Vigor Firmware Container
```
Comment (rhelmke) [inline fact_helper_file/mime/custom_mime_firmware_containers:720]: What about the samples we saw from other draytek devices? Is `2RDH` really a universal magic word for all kind of draytek vigor firmware containers?

**Review by rhelmke — CHANGES_REQUESTED** [review]

File: fact_helper_file/mime/custom_mime_firmware_containers (hunk 2 above)
Comment (LukasEpheser) [reply fact_helper_file/mime/custom_mime_firmware_containers]: For now this mime type only covers the Draytek Vigor 167 modem. Specified description in c4c6341d9cfc83e9b4e61e887ec3c2cb98aa8dc4

**Review by rhelmke — APPROVED** [review]
Thanks! Looks good to me now!

Dispatching tests.

**Comment by rhelmke** [comment]
Great, lets merge!

**Pull request merged**